mlora/executor/task/train_task.py

```diff
@@ -96,13 +97,21 @@ def _save(self, dir_suffix: str = "", additional_info: Dict[str, str] = {}):
         adapter_config["base_model_name_or_path"] = self.llm_name_
         adapter_config = {**adapter_config, **additional_info}
         adapter_config = {**adapter_config, **self.config_.adapter_.export()}
+        logging.info(f"save optimizer weight")
```

Comment:
the log maybe confuse, which optimizer be saved.

Reply:
ok,i will modify later